src/layouts/MainLayout.js

```diff
@@ -18,19 +22,100 @@ const MainLayout = ({ children }) => {
     }
   `);
 
+  const [isMobileNavOpen, setIsMobileNavOpen] = useState(false);
+  const location = useLocation();
+  const isSmallScreen = useMedia('(max-width: 760px)');
+
+  useEffect(() => {
+    setIsMobileNavOpen(false);
+  }, [location.pathname]);
+
   return (
-    <>
+    <div
+      css={css`
+        --global-header-height: 36px;
+        --sidebar-width: 300px;
+
+        min-height: 100vh;
+      `}
+    >
       <GlobalHeader />
-      <main
+      <div
```

Comment:
Out of curiosity, why did you update `main` to be `div`? Wouldn't we want to use the more semantic option (even if it means adding an extra `display: block;`)?

Reply:
Per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) (emphasis mine):

> The content of a <main> element should be unique to the document. Content that is repeated across a set of documents or document sections **such as sidebars**, navigation links, copyright information, site logos, and search forms **shouldn't be included** unless the search form is the main function of the page.

Since this `div` includes the sidebar, I moved the `main` tag to be the actual page content itself. You'll see it used on [line 95](https://github.com/newrelic/docs-website/pull/161/files#diff-a80edad99cd274f77da55f52c521689a32127055d855e00ee2f0f33542069bf2R95)